recipe/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "pytest-shard" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b86a967fbfd1c8e50295095ccda031b7e890862ee06531d5142844f4c1d1cd67 
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
+  run:
+    - pytest
+    - python
+
+test:
+  imports:
+    - pytest_shard
+  requires:
+    - pip
+    - ripgrep
+  commands:
+    - pip check
+    - pytest --traceconfig | rg pytest-shard-{{ version }}
+
+about:
+  home: "https://github.com/AdamGleave/pytest-shard"
+  license: MIT
+  license_family: MIT
+  license_url: https://github.com/AdamGleave/pytest-shard/blob/64610a08dac6b0511b6d51cf895d0e1040d162ad/LICENSE
+  summary: "Shards tests based on a hash of their test name."
```

```suggestion
  summary: Shards tests based on a hash of their test name.
```
No need for double quotes as it's already a string